asreview/feature_extraction/tfidf.py

```diff
@@ -9,21 +7,26 @@ class Tfidf(BaseFeatureExtraction):
     """Class to apply SKLearn Tf-idf to texts."""
     name = "tfidf"
 
-    def __init__(self, *args, ngram_max=1, **kwargs):
+    def __init__(self, *args, ngram_max=1, stop_words=0, **kwargs):
         """Initialize tfidf class.
 
         Arguments
         ---------
         ngram_max: int
             Can use up to ngrams up to ngram_max. For example in the case of
             ngram_max=2, monograms and bigrams could be used.
+        stop_words: int
+            Use *ENGLISH* stop words.
         """
         super(Tfidf, self).__init__(*args, **kwargs)
         self.ngram_max = ngram_max
-        self._model = Pipeline([
-            ('vect', CountVectorizer(ngram_range=(1, ngram_max))),
-            ('tfidf', TfidfTransformer())]
-        )
+        self.stop_words = stop_words
+        if stop_words == 1:
```

Comment:
Why not using the arguments like in sklearn? So 'english' instead of 1?

Reply:
This is easier for hyper parameter optimization.

Reply:
I would like to have the API as flexible as possible. Can we move this to the hyper parameter optimization scripts? That would be great.

Reply:
I'm not sure if it's that simple. What is perhaps possible is to support ["english", "none", None]. It would be practically similar to the SKLearn API, and might cause less breakage (no guarantees).